docs/TOFS_pattern.rst

```diff
@@ -0,0 +1,518 @@
+.. _tofs_pattern:
```

Comment:
This file should be removed since TOFS isn't being used in this formula (the `files_switch` macro isn't imported anywhere).